homeassistant/helpers/util.py

```diff
@@ -0,0 +1,31 @@
+"""Utils around HomeAssistant."""
+from pathlib
+import tempfile
+
+
+def secure_path_check(hass, path):
+    """Check if a path is valid for send external.
+
+    Valid are:
+    - HASS_CONFIG/output
+    - HASS_CONFIG/www
+    - SYSTEM_TEMP
+
+    Return True if valid or False.
+    """
+    try:
+        parts = pathlib.Path(path).resolve().parent().parts()
```

Comment:
I think that it's easier to call absolute path and then just compare the start of the string